.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+name: release
+
+on:
+  push:
+    tags:
+      - obal-[0-9]+.[0-9]+.[0-9]+
```

Comment:
Since it's a regex, should `.` be escaped?

Reply:
it's not a regex. https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet